### ISSUE TITLE
Items property adding

### ISSUE THREAD

**Issue by Drakaden** [zunath/CyberpunkZombieSurvival_JVM#154]
Collin ands I been discussing the t2+ gear in game and shown me a few pieces in the work,
It may be a good plan to add more properties than just the firearm ones when you get to add them, like mana, health and all the other little things that can be bonus or penalty, so we have a lot of flexibility on the future gears.

**Comment by Drakaden** [comment]
Brainstorming time!

Mana: It can be either amount of mana points, or raise or lower the Mana stat allocation rank
Health: Same as Mana
Natural Regeneration: Like casting speed, set at % bonus or penalty
Firearms are already an idea in the work so i won't go into that unless needed
Item Repair and Lockpicking - I'm not sure if we have any properties for these 2, i know lock picking exist as a default nwn skill, but we're talking custom skills there.
Researching - I would see this on rare items, i never really liked the idea of having to invest SP into that stat, alternatives discussion open?

**Comment by taelon875** [comment]
I Say we leave everything The same as point point in allocation rank, it will be the easiest that way. In design and use. 
As for researching I think that can be a stat we omit when adding, I don't ever see ti being used on the gear, but I suppose it could just be added to be there if it is not all that much of a problem.

**Comment by Drakaden** [comment]
It's an old idea of a year ago, but perhaps it can be good to be used, Research could be linked to story quests in the game, these would grant a research point, and you get a certain amount of pts depending on what tier you quest in, this way you would unlock research pts without having to spend SP and it would be linked to tiered areas, thoughts?

**Comment by Drakaden** [comment]
Oh also Reload Speed, if that can be done, reloading bonus/penalty speed

**Comment by zunath** [comment]
These are good ideas which I'll get to at some point. 

Re: Researching - I've never liked it being in the skill allocation menu. But it didn't fit well as a "use it and get better" skill, since it's a passive process. I like the idea of earning them as quest rewards though.

**Comment by taelon875** [comment]
I agree with them being quest rewards. Or even triggering a point up when you enter certain areas.
